semver/greater_than_range_test.ts

```diff
@@ -167,3 +168,43 @@ Deno.test("greaterThanRange() checks if the semver is greater than the range", a
     });
   }
 });
+
+Deno.test("greaterThanRange() handles equals operator", () => {
+  const version = {
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  };
+  const range = [[{
+    operator: "=" as unknown as Operator,
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  }]];
+  assertEquals(greaterThanRange(version, range), false);
+});
+
+// BUG `!=` operator type does not exist in semver and should be removed.
```

Comment:
I don't consider `!=` itself as bug. It's already shipped as stable API. I think we shouldn't remove it at least during `@std/semver@1.x` versions.

If we try to remove it, we first need to discuss it with more people (included both members from core team and community) and need to make consensus to make that breaking change at `@std/semver@2.0.0`